tests/plugins/test_koji_import.py

```diff
@@ -622,6 +622,39 @@ def test_koji_import_wrong_source_type(self, tmpdir, os_env):
             runner.run()
         assert "plugin 'koji_import' raised an exception: RuntimeError" in str(exc)
 
+    @pytest.mark.parametrize(('isolated'), [
+        False,
+        True,
+        None
+    ])
+    def test_isolated_metadata_json(self, tmpdir, monkeypatch, os_env, isolated):
+        session = MockedClientSession('')
+        tasker, workflow = mock_environment(tmpdir,
+                                            session=session,
+                                            name='ns/name',
+                                            version='1.0',
+                                            release='1')
+        runner = create_runner(tasker, workflow)
+
+        monkeypatch.setenv("BUILD", json.dumps({
+            'metadata': {
+                'creationTimestamp': '2015-07-27T09:24:00Z',
+                'namespace': NAMESPACE,
+                'name': BUILD_ID,
+                'labels': {
+                    'isolated': isolated
```

Comment:
`isolated` should not be set if its `None`. Instead the test should set this value only if `isolated` is not `None`